db/post_migrate/20211206160212_drop_token_from_email_tokens.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class DropTokenFromEmailTokens < ActiveRecord::Migration[6.1]
+  DROPPED_COLUMNS ||= {
+    email_tokens: %i{token}
+  }
+
+  def up
+    DROPPED_COLUMNS.each do |table, columns|
+      Migration::ColumnDropper.execute_drop(table, columns)
+    end
```

Comment:
this indirection and constant feels a bit weird. Simply: 

```
Migration::ColumnDropper.execute_drop('email_tokens', ['token'])
```

?

Reply:
Unfortunately we need that constant, otherwise restoring a backup might fail due to missing functions in the `discourse_functions` schema.

https://github.com/discourse/discourse/blob/d2c5165052fd8ef588bb16523d616bb2732ec379/lib/backup_restore/database_restorer.rb#L156-L180